com.unity.ml-agents/Runtime/Policy/BehaviorParameters.cs

```diff
@@ -30,7 +30,14 @@ enum BehaviorType
         InferenceDevice m_InferenceDevice;
         [HideInInspector]
         [SerializeField]
+
+        // Disable warning /com.unity.ml-agents/Runtime/Policy/BehaviorParameters.cs(34,22):
```

Comment:
minor: can we remove the line numbers "(34,22)" as I presume those aren't relevant (and are now incorrect)